shablona/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .shablona import *
-from .version import __version__
\ No newline at end of file
+from .shablona import *  # noqa
```

Comment:
It means ignore the line for pep8/pyflakes (no quality assurance, I guess). pyflakes complains if you import things without explicitly using them. It's great at finding unused imports, variables, and variables used for assignment, but it complains with `from x import *` and things like this.